services/graphql-server/src/graphql/definitions/google-data-api/youtube.js

```diff
@@ -2,6 +2,12 @@ const gql = require('graphql-tag');
 
 module.exports = gql`
 
+extend type Query {
+  validateYoutubePlaylistId(input: String!): Boolean!
```

Comment:
These should all accept object inputs for consistency (and to allow for additional input values in the future)